docs/backends/pyspark.qmd

```diff
@@ -81,7 +79,7 @@ con = ibis.pyspark.connect(session=session)
 :::
 
 ::: {.callout-note}
-The `pyspark` backend does not create `SparkSession` objects, you must create a `SparkSession` and pass that to `ibis.pyspark.connect`.
```

Comment:
Technically it can, though, using `from _url`...

Reply:
Yeah, this document is bit outdated.